### DISCUSSION TITLE
Copilot autocomplete not working inside LunarVim, works fine in basic Neovim

### DISCUSSION THREAD

**Discussion by Shookit** [community/community#40858] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I'm trying to get the github/copilot.vim plugin to work inside of LunarVim, but I'm struggling to get any recommendations from the Copilot engine. I suspect that the LunarVim autocomplete system is somehow conflicting with the Copilot plugin, but I'm coming up empty while digging into this, and I was hoping this could maybe help out someone else.

Of note, when using vanilla neovim, the plugin works totally fine (and again, I'm wondering if it's because it's because I don't have any autocomplete systems that might conflict). Has anyone gotten this working in their Lunar setup?

**Comment by ossanna16** [comment]
Hi there @Shookit and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by jroes** [comment]
Relevant reddit thread here that might help: https://www.reddit.com/r/neovim/comments/sk70rk/using_github_copilot_in_neovim_tab_map_has_been/

I’m pretty sure LunarVim is using cmp, which is overriding tab.

It sounds like you’re having trouble with the automatic completions though, which I’m also experiencing and unsure how to resolve yet. But, hope this helps a little!

**Comment by jroes** [comment]
FWIW, I followed the steps above and the auto-completion is working now (without hitting tab yet). I would give it a try!
